features/flows/rest/impl/src/main/java/org/opennms/netmgt/flows/rest/internal/classification/ClassificationRestServiceImpl.java

```diff
@@ -183,6 +183,18 @@ public Response updateRule(int id, RuleDTO newValue) {
         return Response.ok(convert(rule)).build();
     }
 
+    @Override
+    public Response patchPositionOfRule(int id, RuleDTO patch) {
+        final Rule rule = classificationService.getRule(id);
```

Comment:
The way it is implemented now, `id` will always be `0`.

Reply:
as discussed on the phone => we removed this method completely and will use updateRule instead